src/doc/trpl/ffi.md

```diff
@@ -544,8 +545,8 @@ pub extern fn oh_no() -> ! {
 # fn main() {}
 ```
 
-If you’re writing code that may panic, you should run it in another thread,
-so that the panic doesn’t bubble up to C:
+But it could do anything. If you’re writing code that may panic, you should run
```

Comment:
I find this kinda confusing that this starts out by saying that the code could abort but then follows it up by saying it could do anything instead. Overall there's not a great thing we can say here without a stable `catch_panic`. I think the core truth here is that unwinding just shouldn't cross FFI boundaries, and the way to do that is `catch_panic`, once stabilized.